Earthworks-01/readme.md

```diff
@@ -0,0 +1,28 @@
+
+### Intent
+
+This unit test establishes a simple embankment of infrastructure facilities.
+
+### Prerequisites
+
+This scenario builds upon following other scenarios:
+
+- ProjectSetup-1
+- Tin-1
+
+### Content
+
+This scenario covers the following concepts and/or IFC entities:
+
+- IfcEarthworksFill
+- IfcEarthworksFillTypeEnum
+
+### Supporting files
+
+Following files correspond to this scenario:
+
+| Filename             | Description                                                  |
+| -------------------- | ------------------------------------------------------------ |
+| UT-Earthworks-01.ifc | IFC file with content                                        |
+| UT-Earthworks-01.png | picture showing the TIN surface and embankment               |
+| UT-Earthworks-01.dwg | drawing file with BricsCAD TIN Surface<br/>and Grading representing embankment |
```

Comment:
Similarly for file names, please use the grave accent escape character.

Reply:
Added